### PR TITLE
Add Zigbee2MQTT icon

### LINKED ISSUE #7766: Request: Zigbee2mqtt
### Brand Name

Zigbee2mqtt

### Website

https://www.zigbee2mqtt.io/

### Popularity Metric

gitHub stars is 8,5k

### Official Resources for Icon and Color

https://github.com/Koenkk/zigbee2mqtt/blob/master/images/logo_vector.svg

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Zigbee2MQTT

**Issue:** closes #7766

**Similarweb rank:** [83607](https://similarweb.com/website/zigbee2mqtt.io)

### Checklist
- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
"You've had 2 weeks off, your hands barely work, and you can't see straight; this looks like a nice easy icon to get you back into the swing of things before you start reviewing PRs again."

_**WRONG!**_ The [source file](https://github.com/Koenkk/zigbee2mqtt/blob/434981b567122c745c6a5228cd89b74694b15e4c/images/logo_vector.svg) is an absolute mess!

Colour from [website](https://www.zigbee2mqtt.io/) stylesheet (`--c-brand`).

### CODE REVIEW

File: icons/zigbee2mqtt.svg
Comment (adamrusted) [inline icons/zigbee2mqtt.svg:1]: I would probably argue for inverting this - so keeping the fill on the octogon, and cutting the inner shapes out - for clarity sake. See below image, what do you think @PeterShaggyNoble?

image

Comment (PratyushJayachandran) [reply icons/zigbee2mqtt.svg:1]: I agree with @adamrusted.
However, do you have a better suggestion @Koenkk?

Comment (Koenkk) [reply icons/zigbee2mqtt.svg:1]: I'm fine, but why not keep the inner shapes black?

Comment (adamrusted) [reply icons/zigbee2mqtt.svg:1]: Unfortunately @Koenkk we only deal in monochrome icons on this project - so it'd have to be a single shape with cut-outs. It's up to users of this project then how they display the icon.

Comment (Koenkk) [reply icons/zigbee2mqtt.svg:1]: I see, then this is fine.

**Review by PratyushJayachandran — APPROVED** [review]
I see that you have did some corrections from the source @PeterShaggyNoble. I feel it looks good 😄

**Review by adamrusted — CHANGES_REQUESTED** [review]
image

Unsure if this is just Affinity Designer playing games with me - but I'm still seeing these differences!

**Review by adamrusted — CHANGES_REQUESTED** [review]
Seeing different differences in Illustrator - but they're still present!
Would you mind taking another look please @PeterShaggyNoble?

**EDIT:** If you include the stroke (I haven't, every time) it's only _slightly_ off in the centre there.

image

**Comment by PeterShaggyNoble** [comment]
Took one more pass at it from scratch. The nested SVGs with differing `viewBox`es don't play nice with Inkscape at all but this version _should_ line up a little better.

**Review by PratyushJayachandran — APPROVED** [review]
I am okay with this rendition as the original source is not the cleanest!

**Comment by PeterShaggyNoble** [comment]
Is this good to merge, so @PratyushJayachandran & @adamrusted?

**Pull request merged**